### PR TITLE
Update homeworks

### PR DESCRIPTION
**Pull request by francois-rozet**
* I have fixed the typos in the 1st homework.
* I still have to check that the 2nd one actually runs on GPU. By the way, there is a problem with the CIFAR10 dataset certificate which prevents it from being downloaded. I have fixed it by disabling the certificate check according to [this issue](https://github.com/pytorch/vision/issues/5039).

### CODE REVIEW

File: homeworks/homework2.ipynb [hunk 1]
```diff
@@ -204,9 +224,9 @@
    "source": [
     "### 1.2 Running operations on a GPU\n",
     "\n",
-    "As you may know, one important aspect of deep learning is that large models can be trained efficiently on specialized hardwares such as Graphical Processing Units (GPUs) or Tensorial Processing Units (TPUs). Pytorch allows you to perform operations on GPUs very easily by simply transfering the concerned models and/or tensors to the GPU.\n",
+    "As you may know, one important aspect of deep learning is that large models can be trained efficiently on specialized hardwares such as Graphical Processing Units (GPUs) or Tensorial Processing Units (TPUs). PyTorch allows you to perform operations on GPUs very easily by transfering the concerned models and/or tensors to GPUs.\n",
```
Comment (yannclaes) [inline homeworks/homework2.ipynb]: transfering -> transferring

File: homeworks/homework2.ipynb [hunk 2]
```diff
@@ -352,9 +355,9 @@
    "source": [
     "## 2.  Classifying the CIFAR10 dataset with an MLP\n",
     "\n",
-    "Now that you know how to handle datasets, we are ready to properly train today's first deep learning model on the CIFAR10 dataset. Before we dive into it, **do not underestimate** the importance of properly pre-processing the data before training neural networks. This step is as important as defining the neural architectures themselves but is very often overlooked.\n",
+    "Now that you know how to handle datasets, we are ready to properly train today's first deep learning model on the CIFAR10 dataset. Before we dive into it, **do not underestimate** the importance of properly pre-processing the data before training neural networks. This step is as important as defining the neural architectures themselvesm, but is very often overlooked.\n",
```
Comment (yannclaes) [inline homeworks/homework2.ipynb]: themselvesm -> themselves

File: homeworks/homework2.ipynb [hunk 3]
```diff
@@ -352,9 +355,9 @@
...
+    "In this exercise you are provided with an already defined multi-layer perceptron that you can train to classify CIFAR10 images. The structure of the network is already defined, yet some crucial hyperparameters are missing. It is your job to fill them in and successfully train the network. As part of the exercise, you are also required to monitor the evolution of training: this usually consists in checking how the training and testing losses evolve during training and keeping track of the model's accuracy on the testing set. Report these statistics with some plots. In addition, transfer the network and the mini-batches on GPU to speed up the training.\n",
```
Comment (yannclaes) [inline homeworks/homework2.ipynb]: speed up training

File: homeworks/homework2.ipynb [hunk 4]
```diff
@@ -367,11 +370,11 @@
    "source": [
     "input_features =\n",
     "output_features = \n",
-    "hidden_feautres =  # your code\n",
+    "hidden_feautres = # your code\n",
```
Comment (yannclaes) [inline homeworks/homework2.ipynb]: hidden_feautres -> hidden_features

File: homeworks/homework2.ipynb [hunk 5]
```diff
@@ -367,11 +370,11 @@
...
     "learning_rate =\n",
     "num_epochs =\n",
     "\n",
-    "class net(nn.Sequential):\n",
+    "class MLP(nn.Sequential):\n",
     "    def __init__(self, input_features, output_features, hidden_feautres):\n",
```
Comment (yannclaes) [inline homeworks/homework2.ipynb]: hidden_feautres -> hidden_features

File: homeworks/homework2.ipynb [hunk 6]
```diff
@@ -384,7 +387,10 @@
     "            nn.Linear(hidden_feautres, output_features),\n",
```
Comment (yannclaes) [inline homeworks/homework2.ipynb]: hidden_feautres -> hidden_features

File: homeworks/homework2.ipynb [hunk 7]
```diff
@@ -6,7 +6,7 @@
    "source": [
     "# INFO8010: Homework 2\n",
     "\n",
-    "Last week you learned how to program your first neural network starting from the very first principles of deep learning. If you managed to solve last week's assignment without any problems **congratulations!**. If that was not the case **don't worry**, here's a second assignment for you which you can use to get better at deep learning.\n",
+    "Last week you learned how to program your first neural network starting from the very first principles of deep learning. If you managed to solve last week's assignment without any problems **congratulations!** If that was not the case **don't worry**, here's a second assignment for you which you can use to get better at deep learning.\n",
```
Comment (lgaspard) [inline homeworks/homework2.ipynb]: "Last week" -> "Two weeks ago" or "Last time" or "In the previous assignment"
"last week's assignment" -> "the first assignment"

**Pull request merged**